index.js

```diff
@@ -184,7 +185,13 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
+
+    if (resolveConfig.root) {
+      resolveConfig.modules = [].concat(resolveConfig.root);
```

Comment:
I'll update to make sure `node_modules` is always included. Better safe than sorry :)